simbad/command_line/simbad_database.py

```diff
@@ -31,11 +31,6 @@
 
 from simbad.util.pdb_util import get_pdb_content
 
-try:
-    import morda
```

Comment:
This important is required. Andrey uses it to initialise MoRDa and all environment variables 🤦‍♂️ . Until this changes, we are required to keep the import in. The `try-except` clause is in-place if MoRDa (Python package) is not installed.

Reply:
We already import morda without the `try-except` clause. Morda went into ccp4 before simbad so I don't see a reason to have a `try-except` clause